src/OrganisationRegistry.ElasticSearch.Projections/Body/BodyHandler.cs

```diff
@@ -98,11 +97,9 @@ private void PrepareIndex(IElasticClient client, bool deleteIndex)
 
             if (!client.DoesIndexExist(indexName))
             {
-                var indexResult = client.CreateIndex(
+                var indexResult = client.Indices.Create(
                     indexName,
-                    index => index
-                        .Mappings(mappings => mappings
-                            .Map<BodyDocument>(typeName, BodyDocument.Mapping)));
+                    index => index.Map<BodyDocument>(BodyDocument.Mapping));
```

Comment:
Just wondering, the `typeName` that is not here anymore, does that have any impact?